credsweeper/common/morpheme_checklist.txt

```diff
@@ -0,0 +1,1326 @@
+000
+111
+222
+333
+444
```

Comment:
Could you explain this `morpheme_checklist.txt` file came from?

Reply:
The list was created manually after false positive cases review.
The approach is: 
- got a value which looks like a token "PKGVULNERABILITY"
- split long word to morphemes on English words which might be present in text "VULN" (a subset of vulnerable) and "ABILITY" (the word and postfix). 3 letter words may be acceptable if the sequence is a word.
- add the 2 morphemes to list
- run test to be sure it does not removed many others
- run scan and research next FP.
- 3 digits were found too. In real token the sequence has low probability, so i added it too.

Reply:
Thank you for explanation, next time it would be good to add rules that using those filters.